src/Service/Resolvers/SqlPaginationUtil.cs

```diff
@@ -555,4 +554,4 @@ public static string FormatQueryString(NameValueCollection queryParameters)
             return queryString;
         }
     }
-}
+}
```

Comment:
```suggestion
}

```
new line at end of file